scw/errors.go

```diff
@@ -44,18 +57,12 @@ func hasResponseError(res *http.Response) error {
 }
 
 func (e *ResponseError) Error() string {
-	s := fmt.Sprintf("received status code %d", e.StatusCode)
+	s := fmt.Sprintf("http request [%s %s] return [%s]", e.RequestMethod, e.RequestUrl, e.Status)
 
-	if e.Type != "" {
-		s = fmt.Sprintf("%s: error type is %s", s, e.Type)
+	jsonStr, _ := json.MarshalIndent(e, "", "  ")
+	if len(jsonStr) > 0 {
+		s = fmt.Sprintf("%s\n%s", s, jsonStr)
```

Comment:
an error should not be multi-line, maybe use a log.Debugf in `hasResponseError` to show print full http body